proposals/0021/README.md

```diff
@@ -1,30 +1,212 @@
+digest: String that uniquely identifies this package version (optional)
+license: SPDX identifier of the package license (https://spdx.org/licenses/) (optional)
+homeURL: The URL of the project home page (optional)
+appVersion: The version of the app that this contains (optional)
+containersImages: # (optional)
+  - name: Image identifier (optional)
+    image: The format should match ${REGISTRYHOST}/${USERNAME}/${NAME}:${TAG}
+    whitelisted: When set to true, this image won't be scanned for security vulnerabilities
+containsSecurityUpdates: Whether this package version contains security updates (optional, boolean)
+operator: Whether this package is an Operator (optional, boolean)
+deprecated: Whether this package is deprecated (optional, boolean)
+keywords: # (optional)
+  - A list of keywords about this package
+  - Using one or more categories names as keywords will improve package visibility
+links: # (optional)
+  - name: Title of the link (required for each link)
+    url: URL of the link (required for each link)
+readme: | # (optional)
+  Package documentation in markdown format
+
+  Content added here will be rendered on Artifact Hub
+install: | # (optional)
+  Brief install instructions in markdown format
+
+  Content added here will be displayed when the INSTALL button on the package details page is clicked.
+changes: # (optional)
+  - A list of changes introduced in this package version
+  - Use one entry for each of them
+maintainers: # (optional)
+  - name: The maintainer name (required for each maintainer)
+    email: The maintainer email (required for each maintainer)
+provider: # (optional)
+  name: The name of the individual, company, or service that provides this package (optional)
+ignore: # (optional, entries use .gitignore syntax)
+  - lib
+```
+
+https://github.com/artifacthub/hub/blob/master/docs/metadata/artifacthub-pkg.yml
+
+Having a clear separation between the metadata used by ArtifactHub and the
+actual artifact (the image itself) will sound a complication, but it gives us
+more control and flexibility between ongoing work (PRs on specific actions) and
+their release cycle (ActionHub artifact).
+
 ## System-context-diagram
 
+A contributor will interface with a traditional repository, currently located in
+[github.com/gianarb/hub](https://github.com/gianarb/hub), it will be moved to its
+permanent location [github.com/gianarb/hub](https://github.com/tinkerbell/hub).
+
+I will explain the layout using the command `tree` and commenting various
+directories:
+
+```terminal
+./hub $ tree -L 2
+.
+├── Makefile
+├── README.md
+    # The actions directories contains a subfolder for every aciton, in this
+    # case we have only one called disk-wipe
+├── actions
+│   └── disk-wipe
+    # This directory is not ment to be modified by human. It contains generated
+    # manifests from the action directory. Those manifests are periodically
+    # scraped by ArtifactHub and used to populate their webisite.
```

Comment:
this comment is out of place right? its meant to be under `artifacthub-manifests` right?
Also, there's a typo ment -> meant.

Reply:
I removed the directory becuase ArtifactHub is adding support for scraping a particular branch, and that was my original idea, in this way the manifests are not where the actions are and people can't do mistakes manually updating them :P